apps/blog-app/src/content/2026-08-06-when-the-eyeball-left-the-loop.md

```
@@ -0,0 +1,250 @@
+the step back, explicitly and with instruments. Step through the three stages:
+
+<div data-chart="loop">Diagram: the verification step moving from an implicit, continuous human glance (old loop) to nothing at all (naive delegation) to a named, numeric VERIFY stage (working loop). Enable JavaScript for the interactive walkthrough.</div>
+
+The shift in one line: **you stopped being the author and became the instrument.**
+
+## Why code review can't cover the gap
+
+Between what an agent writes and what a user sees are layers that only exist at
+runtime:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)